diff --git a/setup.py b/setup.py
index 6da2d4d..32262dd 100644
--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@ setup(
     packages=['privacypass', 'privacypass.tests'],
     zip_safe=False,
     platforms='any',
-    setup_requires=['milksnake', 'setuptools_scm'],
+    setup_requires=['milksnake'],
     install_requires=['cffi', 'attrs'],
     extras_require={
         "tests": [
@@ -29,7 +29,6 @@ setup(
             "hypothesis",
         ],
     },
-    use_scm_version=True,
     url='https://github.com/',
     milksnake_tasks=[
         build_native
